setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
+sudo apt -y install build-essential autoconf autoconf-archive automake m4 libtool gcc pkg-config libqrencode-dev pandoc doxygen liboath-dev iproute2 plymouth libplymouth-dev libssl-dev libjson-c-dev libcurl4-openssl-dev
+
+mkdir ./tpm2-totp
+cd tpm2-totp
+
+git clone https://github.com/tpm2-software/tpm2-tss
+cd tpm2-tss
+./bootstrap
+./configure
+make 
+sudo make install
+cd ..
+
+git clone https://github.com/tpm2-software/tpm2-totp
+cd tpm2-totp
+./bootstrap
+./configure
+make
+sudo make install
+
+sudo ldconfig
```

Comment:
Not super self-explanatory.

Reply:
basically the TPM2-TOTP module gets installed and has to be loaded before it can be used.

Reply:
A comment to this effect would be nice is all I meant 😄

Reply:
I will memorialize this as the one time @eventualbuddha asked me to add a comment, rather than take one away ;)